os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -357,10 +354,19 @@ static int loading_thread(int argc, char *argv[])
 		return -1;
 	}
 
+	/* Arguments : [1] type */
 	type = (pid_t)atoi(argv[1]);
```

Comment:
type is defined as int type and here it is being typecasted as pid_t. Please change the variable type accordingly.